src/Compilers/Core/Portable/InternalUtilities/ConcurrentBag.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.InternalUtilities
+{
+    internal class ConcurrentBag<T>
+        : System.Collections.Concurrent.ConcurrentBag<T>, ICollection<T>
+    {
+        public bool IsReadOnly
+        {
+            get
+            {
+                throw new NotImplementedException();
```

Comment:
Are these not implemented because you haven't gotten around to it yet, or because they shouldn't be callable?